todo_app/app.py

```diff
@@ -1,11 +1,25 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect,url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import add_item, get_items
 
 app = Flask(__name__)
 app.config.from_object(Config())
-
+    
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    #return 'Hello World!'
```

Comment:
Don't  be afraid to delete code rather than commenting it out - commented out code quickly builds up to make it harder to read your code, and deleted code can always be recovered with Git if we do want it at a later time